src/util.cpp

```diff
@@ -173,6 +173,10 @@ bool Util::MatchMod(const char *match, const char *mod, double *output) {
     return !*pmatch && !*pmod;
 }
 
+void Util::Capitalise(std::string &str) {
```

Comment:
Don't modify it in place, return a new string instead.

Reply:
Again IIRC this was following the style of other (Util) functions, but sure, can do.